src/main/kotlin/com/kotlin/boardproject/domain/post/dto/normalpost/NormalPostByQueryResponseDto.kt

```diff
@@ -1,8 +1,8 @@
-package com.kotlin.boardproject.dto.post.normalpost
+package com.kotlin.boardproject.domain.post.dto.normalpost
 
-import com.kotlin.boardproject.dto.post.normalpost.NormalPostByQueryElementDto.Companion.fromNormalPostToQueryOneNormalPostResponseDto
-import com.kotlin.boardproject.model.NormalPost
-import com.kotlin.boardproject.model.User
+import com.kotlin.boardproject.domain.post.dto.normalpost.NormalPostByQueryElementDto.Companion.fromNormalPostToQueryOneNormalPostResponseDto
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end